react/features/overlay/components/AbstractPageReloadOverlay.js

```diff
@@ -14,7 +14,7 @@ import {
     isFatalJitsiConnectionError
 } from '../../base/lib-jitsi-meet';
 
-import ReloadButton from './ReloadButton';
```

Comment:
you sure this doesn't run on mobile?

Reply:
No, and it has a web render method (with divs and all)

Reply:
Btw it's not good that we have it referenced here in the abstract class, but I just really didn't want to push this PR as far as fixing even this